src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js

```diff
@@ -27,6 +27,7 @@ class List extends React.Component { // eslint-disable-line react/prefer-statele
                 <th className="">Subcategorie</th>
                 <th className="">Afdeling</th>
                 <th className="">Status</th>
+                <th className="">Urgentie</th>
```

Comment:
Even on a large monitor, the table does not fit correctly in the current design, I think we should fix that. Maybe by allowing the container/table to have a larger width?

Reply:
ja goed punt ik ga ernaar kijken